### PR TITLE
feat: adding index for updated at ascending for asks endpoint

### PR DESCRIPTION
**Pull request by JadenDurnford**

### CODE REVIEW

File: packages/indexer/src/migrations/1643809164129_orders.sql [hunk 1]
```diff
@@ -131,6 +131,10 @@ ALTER TABLE "orders" SET (
   autovacuum_analyze_threshold=100000
 );
 
+-- CREATE INDEX "orders_asks_updated_at_asc_id_index"
```
Comment (ipeleg) [inline packages/indexer/src/migrations/1643809164129_orders.sql:134]: What is this for? and i guess if we want it we want to be uncommented right?

Comment (JadenDurnford) [reply packages/indexer/src/migrations/1643809164129_orders.sql:134]: We created this index for the lightnode, @nofir should this be uncommented now that it has been successfully created?

**Review by nofir — DISMISSED** [review]

File: packages/indexer/src/migrations/1643809164129_orders.sql (hunk 1 above)
Comment (ipeleg) [inline packages/indexer/src/migrations/1643809164129_orders.sql:134]: @JadenDurnford as it's now in a migration file that was already executed we can uncomment this

**Review by ipeleg — CHANGES_REQUESTED** [review]

File: packages/indexer/src/migrations/1643809164129_orders.sql (hunk 1 above)
Comment (nofir) [reply packages/indexer/src/migrations/1643809164129_orders.sql]: @JadenDurnford ust notice that we will have to run it manually in polygon, etc

**Review by nofir — APPROVED** [review]

**Review by ipeleg — APPROVED** [review]

**Pull request merged**